spec/models/weekly_email_volume_spec.rb

```diff
@@ -84,5 +84,16 @@
         expect(described_class.new(person).estimate).to be_nil
       end
     end
+
+    context "given the coronavirus page" do
+      it "returns the estimated volume" do
+        coronavirus_page = {
+          document_type: "topical_event",
+          base_path: "/government/topical-events/coronavirus-covid-19-uk-government-response",
+        }.deep_stringify_keys
+
+        expect(described_class.new(coronavirus_page).estimate).to eql "40 to 60"
+      end
+    end
```

Comment:
Should we add a test when its not the coronavirus page that we aren't showing an estimate?